Zscaler docs (#147)
populationgenomics/team-docs: certificates.md

# Setting up certificates from ZScaler

> As part of the security practices at CPG, we use ZScaler for secure connections from the company devices.

Reviewer: Should we specify that this is not a CPG issue, as much as a MCRI issue. I was a little confused at first @illusional

Reviewer 2: Ah — That could explain why I haven't heard of it!

Author: > Should we specify that this is not a CPG issue, as much as a MCRI issue. I was a little confused at first @illusional 

Apologies guys! I haven't really seen the device setup from CPG so I wasn't aware this was only MCRI specific. Happy to amend if that's more appropriate

Reviewer 3: ```suggestion
As part of the security practices at MCRI and RCH, some devices use [ZScaler](https://www.zscaler.com/company/faqs) for secure connections from company devices.
```
